php/exceptions/NotFound.class.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * This class is an Exception extension created to distinguish any
+ * Exception from the case when a module can't be loaded because it is not a module
+ * (instruments, forms, reliability forms,...)
+ *
+ * PHP Version 5
+ *
+ * @category Errors
+ * @package  Loris
+ * @author   Rida Abou-Haidar <rida.loris@gmail.com>
```

Comment:
My memory is getting bad, i dnt remember authoring this

Reply:
Take the credit and run!